src/EditorFeatures/Test2/FindReferences/FindReferencesTests.LocalFunctions.vb

```diff
@@ -0,0 +1,196 @@
+' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading.Tasks
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.FindReferences
+    Partial Public Class FindReferencesTests
+        <WorkItem(18761, "https://github.com/dotnet/roslyn/issues/18761")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestLocalFunction1() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+class Test
+{
+    void Main()
+    {
+        {
+            int x = 1;
+            [|$$Print|](x);
+            void {|Definition:Print|}(int y) { }
+        }
+
+        {
+            int z = 1;
+            Print(z);
+            void Print(int y) { }
+        }
+    }
+}
+        </Document>
+    </Project>
+</Workspace>
+            Await TestAPIAndFeature(input)
+        End Function
+
+        <WorkItem(18761, "https://github.com/dotnet/roslyn/issues/18761")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestLocalFunction2() As Task
```

Comment:
Can you name these to indicate their difference: `TestLocalFunctionFromDefinition` or something?